oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -691,23 +692,40 @@ int mca_spml_ucx_add_procs(oshmem_group_t* group, size_t nprocs)
         }
     }
 
+    indices = malloc(nprocs * sizeof(*indices));
+    if (!indices) {
```

Comment:
Seems the convention is compare to `NULL`

Reply:
done